packages/table/src/headers/columnHeader.tsx

```diff
@@ -48,24 +48,20 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
         loading: false,
     };
 
-    public shouldComponentUpdate(nextProps: IColumnHeaderProps) {
-        return shouldHeaderComponentUpdate(this.props, nextProps, this.isSelectedRegionRelevant);
-    }
-
     public render() {
         const {
             // from IColumnHeaderProps
-            cellRenderer,
+            cellRenderer: renderHeaderCell,
             onColumnWidthChanged,
 
             // from IColumnWidths
-            minColumnWidth,
-            maxColumnWidth,
+            minColumnWidth: minSize,
+            maxColumnWidth: maxSize,
             defaultColumnWidth,
 
             // from IColumnIndices
-            columnIndexStart,
-            columnIndexEnd,
+            columnIndexStart: indexStart,
+            columnIndexEnd: indexEnd,
```

Comment:
using this syntactic trick everywhere now

Reply:
I love destructuring. So much.